src/layouts/LoggedIn.layout.vue

```diff
@@ -43,4 +43,20 @@ const sidebarExpanded = ref(isDesktop.value);
 const onToggleSidebar = () => {
 	sidebarExpanded.value = !sidebarExpanded.value;
 };
+
+watch(sidebarExpanded, (value) => {
+	localStorage.setItem("sidebarExpanded", value.toString());
+});
+
+watch(isDesktop, (value) => {
+	sidebarExpanded.value = value;
+});
+
+onMounted(() => {
+	const sidebarExpandedValue = localStorage.getItem("sidebarExpanded");
+	if (!sidebarExpandedValue) {
+		localStorage.setItem("sidebarExpanded", "true");
+	}
+	sidebarExpanded.value = sidebarExpandedValue === "true";
+});
```

Comment:
Would it maybe be better to use `useStorage` from `"@vueuse/core"`(https://vueuse.org/core/useStorage/)? 🤔 

I think we could then replace the lines (with line 41 also removed) with
```suggestion
const sidebarExpanded = useStorage("sidebarExpanded", isDesktop.value);

watch(isDesktop, (value) => {
	sidebarExpanded.value = value;
});
```

What do you think about it? 💭